### PR TITLE
Fix "error listing repositories in github" because of rate limiting

### PR DESCRIPTION
**Pull request by mvidner**
provided that you have this in ~/.netrc

    machine api.github.com login your_username password  your_token

where your_token is one managed at https://github.com/settings/tokens

### CODE REVIEW

**Comment by jreidinger** [comment]
and how it will work without `~/.netrc`?

**Comment by lslezak** [comment]
LGTM code wise, but the PR description should be included in the README otherwise this feature is well hidden...

File: README.md [hunk 1]
```diff
@@ -38,6 +38,21 @@ modules
     'FAV' : applies to only your favorite modules, set in ~/.y2m
     <lowercase string> : applies to the named modules
 
+## Authentication
+
+The tool will work even without GitHub authentication, *unless* you share
+an IP with the rest of your company, in which case it will fail because of
+rate limiting. To get an 83 times better rate, authenticate:
+
+Generate an [access token](https://github.com/settings/tokens)
```
Comment (lslezak) [inline README.md]: Um, which scope for the token is needed? I guess just a `public_repo` is enough in this case, isn't it?

And maybe you could directly link to https://github.com/settings/tokens/new ...

**Comment by mvidner** [comment]
Without ~/.netrc it just continues unauthenticated, like before, so that's OK.

Thanks!

**Pull request merged**